com.unity.render-pipelines.core/Editor/CoreEditorDrawers.cs

```diff
@@ -808,6 +808,27 @@ void SwitchEnabler(TData data, Editor owner)
                 ConditionalWithAdditionalProperties((serialized, owner) => additionalPropertiesState[additionalPropertiesMask] && foldoutState[foldoutMask], additionalPropertiesState.GetAnimation(additionalPropertiesMask), additionalContent).Draw
             );
         }
+
+        // IsPreset is an internal API - lets reuse the usable part of this function
+        // 93 is a "magic number" and does not represent a combination of other flags here
+        internal static bool IsPresetEditor(UnityEditor.Editor editor)
+        {
+            return (int)((editor.target as Component).gameObject.hideFlags) == 93;
+        }
+
+        /// <summary>
+        /// Conditioned drawer that will render the normal content if the editor is not editing a preset
+        /// </summary>
+        /// <param name="normalContent">The content of the group</param>
+        /// <param name="presetContent">The content of the group when the editor is preset</param>
+        /// <returns>A IDrawer object</returns>
+        public static IDrawer GroupWithPreset(IDrawer normalContent, IDrawer presetContent)
+        {
+            return Group(
+                    Conditional((s, e) => !IsPresetEditor(e), normalContent),
```

Comment:
what do s and e variables represent here?
also, here we are adding a new public API to have explicit Presets change. is this what we want? isnt it exposing a workaround to our users?

Reply:
Ok , i see that this was a work arround and we do not want to mantain any kind of public api for this